extensions/amp-iframe/1.0/amp-iframe.js

```diff
@@ -31,6 +33,35 @@ class AmpIframe extends BaseElement {
     );
     return super.isLayoutSupported(layout);
   }
+
+  /** @override */
+  init() {
+    return dict({
+      'onLoadCallback': () => {
+        const placeholder = this.getPlaceholder();
+        if (placeholder) {
+          this.togglePlaceholder(false);
+        } else {
+          measureIntersection(this.element).then((intersectionEntry) => {
+            const pos = intersectionEntry.boundingClientRect;
+            const minTop = Math.min(600, this.getViewport().getHeight() * 0.75);
```

Comment:
Discussed offline: Let's use `rootBounds` from the `intersectionEntry` to prevent relying on AMP-specific APIs.